channeldb/db.go

```diff
@@ -276,6 +277,13 @@ var (
 			number:    29,
 			migration: migration29.MigrateChanID,
 		},
+		{
+			// Removes the "sweeper-last-tx" bucket. Migration
+			// version 30 is already used as an optional migration
+			// therefore we use version 31.
+			number:    31,
```

Comment:
We don't have 30 in the `meta` bucket since that is used to save the mandatory migrations, so 29 is in the db atm. Ideally the next version should be 30, but it'd be confusing as hell. Maybe `migration30` is a bad name, should have named it `optional_migration_1`🙈. Anyway, I think as long as the version number increments we should be fine here. Just wanna point it out and double check.

Reply:
Ahh I see, depending on how we decide whether we are going mandatory or optional, I think I will change it to a consecutive number (30) with an explanatory comment.